crypto/src/utils.rs

```diff
@@ -0,0 +1,104 @@
+#![allow(dead_code)]
+use super::{gen_keypair, hash, PublicKey, SecretKey};
+use openssl::symm::{decrypt, encrypt, Cipher};
+use pem::{encode, parse, Pem};
+#[cfg(not(windows))]
+use std::os::unix::fs::OpenOptionsExt;
+use std::{
+    fs::{File, OpenOptions},
+    io::Error,
+    io::Read,
+    io::Write,
+    path::Path,
+};
+const DEFAULT_KEY_PATH: &str = "../private_key_file.pem";
```

Comment:
It seems to me, that this module shouldn't be responsible for settings default key path. I'm going to handle it on the `GenerateNodeConfig` side. And, anyway, as this function is gonna be used for two keys (consensus and service), such default path doesn't make much sense, I think.